web/modules/custom/dpl_library_token/src/LibraryToken.php

```diff
@@ -32,14 +33,16 @@ class LibraryToken {
    *
    * @return LibraryToken
    *   Token object created based on a json formed response.
+   *
+   * @throws \Drupal\dpl_library_token\Exception\LibraryTokenResponseException
```

Comment:
I do not think it is a good idea to add `@throws` annotations everywhere in the project as it stands now

1. We do not have any tooling to support/remind developers to add it in our CI pipeline.
2.  If we add `@throws` we have no way to ensure that it matches the current code. Subsequent can change what exceptions are thrown in a function.
3. The '@throws` section can quickly become littered with exceptions that the developer can do little about except letting them bubble up to the caller. Using [`thecodingmachine/safe`](https://github.com/thecodingmachine/safe) makes this situation worse

It seems like [PHPStan can help us with `@throws` and Exceptions](https://phpstan.org/blog/bring-your-exceptions-under-control) though.

Either I suggest we drop `@throws` entirely ([perhaps even forbid them](https://github.com/slevomat/coding-standard/#slevomatcodingstandardcommentingforbiddenannotations-)) or that we check up on the options provided by PHPStan and form an option on how we want to work with them including checked and unchecked exceptions.